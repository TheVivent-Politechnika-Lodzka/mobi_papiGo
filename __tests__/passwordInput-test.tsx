import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import PasswordInput from '../src/components/passwordInput';

it('Input test', () => {
  const input = renderer
    .create(
      <PasswordInput label="s" defaultValue="as" onChangeText={() => {}} />
    )
    .toJSON();
  expect(input).toMatchSnapshot();
});
