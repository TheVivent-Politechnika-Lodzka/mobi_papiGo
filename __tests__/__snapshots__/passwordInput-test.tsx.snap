// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input test 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#C8E3D4",
      "borderRadius": 20,
      "marginBottom": "20%",
      "paddingHorizontal": 10,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 16,
        "fontWeight": "bold",
        "marginTop": 10,
      }
    }
  >
    s
  </Text>
  <View
    collapsable={false}
    nativeID="animatedComponent"
    style={
      Object {
        "alignItems": "center",
        "borderBottomWidth": 1,
        "borderColor": "#86939e",
        "flexDirection": "row",
        "transform": Array [
          Object {
            "translateX": 0,
          },
        ],
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "height": 40,
          "justifyContent": "center",
          "marginVertical": 4,
          "paddingRight": 4,
        }
      }
    >
      <View
        style={
          Object {
            "overflow": "hidden",
          }
        }
      >
        <View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "justifyContent": "center",
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "#242424",
                    "fontSize": 24,
                  },
                  Object {
                    "backgroundColor": "transparent",
                  },
                  Object {
                    "fontFamily": "Entypo",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              testID="iconIcon"
            >
              
            </Text>
          </View>
        </View>
      </View>
    </View>
    <TextInput
      autoComplete="password"
      defaultValue="as"
      editable={true}
      onChangeText={[Function]}
      placeholder="Hasło"
      placeholderTextColor="#86939e"
      secureTextEntry={true}
      style={
        Object {
          "color": "#242424",
          "flex": 1,
          "fontSize": 18,
          "minHeight": 40,
        }
      }
      testID="RNE__Input__text-input"
      underlineColorAndroid="transparent"
    />
  </View>
  <Text
    style={
      Object {
        "color": "#ff190c",
        "fontSize": 12,
        "margin": 5,
      }
    }
  />
</View>
`;
